.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,7 +7,7 @@ about: Create a report to help reproduce and fix the issue
 
 <!--- **INCLUDE THE ROUND ID**
 If you discovered this issue from playing tgstation hosted servers:
-[Round ID]: # (It can be found in the Status panel or retrieved from https://atlantaned.space/statbus/round.php ! The round id let's us look up valuable information and logs for the round the bug happened.)-->
+[Round ID]: # (It can be found in the Status panel or from the bot in the discord during the round! The round id let's us look up valuable information and logs for the round the bug happened.)-->
```

Comment:
```suggestion
[Round ID]: # (It can be found in the Status panel or from the bot in the discord during the round! The round id let's us look up valuable information and logs for the round the bug happened.)-->
``` 
wtf